factor/totp/lang/en/factor_totp.php

```diff
@@ -38,6 +40,8 @@
 $string['privacy:metadata'] = 'The TOTP Factor plugin does not store any personal data';
 $string['settings:secretlength'] = 'TOTP secret key length';
 $string['settings:secretlength_help'] = 'Generated TOTP secret key string length';
+$string['settings:window'] = 'TOTP verification window';
```

Comment:
can you tweak this to make it clear 1 window = 30 seconds

or maybe even change the drop down so it say 30 sec, 1 min, 2 min, 3 mins, 4 min, 5 min